### PR TITLE
Code split from Matrix.Common

### PR DESCRIPTION
**Pull request by fbsoft**
Split code from Matrix.Common files

### CODE REVIEW

File: GitVersion.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # GitVersion.yml
 mode: ContinuousDeployment
-next-version: 3.3.1
+next-version: 3.3
```
Comment (DmytroLohvynenko) [inline GitVersion.yml:3]: @romaklimenko do we need to adjust this?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -26,26 +26,19 @@ pool:
...
-    @('develop','release','AzurePipelines') | ForEach-Object{
-        dotnet nuget update  source $_ -p $env:NUGET_KEY -u VssSessionToken --configfile nuget.config
-    }
-  condition: and(succeeded(), variables['nuget.key'])
-  displayName: 'Add api token to access nuget artifacts'
-  env:
-    NUGET_KEY: $(nuget.key)
-
-- template: crawler.build.yml@templates
+  - task: NuGetAuthenticate@0
```
Comment (DmytroLohvynenko) [inline azure-pipelines.yml:41]: @romaklimenko not sure about changes. In SqlServer hovewer we have this version.

**Review by DmytroLohvynenko — APPROVED** [review]
Looks good to me

File: azure-pipelines.yml (hunk 2 above)
Comment (romaklimenko) [reply azure-pipelines.yml:41]: It this is the same as in SQL, then we are fine.

**Review by romaklimenko — APPROVED** [review]

**Pull request merged**